### ISSUE TITLE
Alteração de nomes de product types no banco

### ISSUE THREAD

**Issue by Gschwend** [linea-it/pz-server#48]
- [ ] alterar nome: photoz_results -> validation_results (display name: Validation Results) (este mantém no plural)
- [ ] alterar nome:  training_sets  -> training_set (display name:  Training Set) (no singular)      
- [ ] alterar nome:  validation_sets  -> validation_set (display name:  Validation Set) (no singular)      
- [ ] incluir novo product type: photoz_table (display name:  Photo-z Table) (no singular)  
- [ ] incluir coluna de descrição com o conteúdo abaixo:

| name | description|
|---|---|
|specz_catalog         | Catalog of spectroscopic redshifts and positions (usually equatorial coordinates). |
|training_set             | Training set for photo-z algorithms (tabular data). It usually contains magnitudes, errors, and true redshifts.  |
|validation_set          | Validation/test set for photo-z algorithms (tabular data). It usually contains magnitudes, errors, and true redshifts.    | 
| validation_results   | Results of a photo-z validation procedure (free format). Usually contains photo-z estimates (single estimates and/or pdf) of a validation set and photo-z validation metrics. |
| photoz_table          |  Results of a photo-z estimation procedure. If the data is larger than the file upload limit (200MB), the product entry stores only the metadata (instructions on accessing the data should be provided in the description field.  |
